<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleInfo</class>
 <widget class="QDialog" name="SaleInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>从</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>561</width>
     <height>271</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>销售流水号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>商品编码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>单价</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总金额</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>积分</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>销售日期</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="saleNum">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>按日期查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>到</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>销售流水号：</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateBegin">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>36</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEnd">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>35</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
